modules/va_facilities/app/controllers/va_facilities/v1/nearby_controller.rb

```diff
@@ -70,21 +76,34 @@ def validate_params
         validate_type_and_services_known unless params[:type].nil?
       end
 
-      def get_query_method(params)
+      def get_lat_lng(params)
         obs_fields = params.keys.map(&:to_sym)
         location_type = REQUIRED_PARAMS.find do |loc_type, req_field_names|
           no_missing_fields = (req_field_names - obs_fields).empty?
           break loc_type if no_missing_fields
         end
-        QUERY_INFO[location_type]
+
+        lat_lng = params.slice(:lat, :lng)
```

Comment:
We can change the block to look like this:
```suggestion
        if location_type.eql? :address
            GeocodingService.new.query(params[:street_address], params[:city], params[:state], params[:zip])
        else
            params.slice(:lat, :lng)
        end
```
And then remove the `lat_lng` being returned at the end of the method.

Reply:
lol, I changed it to the current implementation because it was discussed that it was clearer to have a final return at the end of the line.

Reply:
That is unnecessary in ruby and not ruby-ish. But if that's not what the team wants, then I'm ok keeping it the way it is

Reply:
I'm going to leave this however it ended up. Six/half-dozen.